src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.Scanner;
+
+public class Duke
+{
+    /* immutable Strings below
+     *
+     *
+     */
+    public static final String GREETING = " Hello! I'm Duke! What can I do for you?";
+
+    public static final String INVALID = " Invalid command!";
+    public static final String GOODBYE = " Bye. Hope to see you again soon!";
+    public static final String LINE_DIVIDER = "____________________________________________________________";
+    public static final int INT = 101;
+
+    /* Storage and variables below
+     *
+     *
+     */
+    private static int indexBaseOne = 1;
+    private static Task[] tasks;
+    private static final Scanner SCANNER = new Scanner (System.in);
+
+
+    public static void main (String[] args)
+    {
+        greet ();
+        String line;
+        initTasks ();
+        line = SCANNER.nextLine ();
+
+        while (true) {
+
+            if (line.equals ("bye"))
+            {
```

Comment:
Perhaps you could check which style of brackets should be used to comply with the coding standards. 

```suggestion
    if (line.equals("bye")) {
        //code here
    }
```